server/src/main/java/org/elasticsearch/discovery/MasterNotDiscoveredException.java

```diff
@@ -27,10 +27,6 @@
 
 public class MasterNotDiscoveredException extends ElasticsearchException {
 
-    public MasterNotDiscoveredException() {
-        super("");
```

Comment:
I wonder whether we should drop the constructor taking a `String` instead, since none of the strings passed are particularly useful.